src/runtime/eng/Versions.props

```diff
@@ -80,6 +80,17 @@
   <PropertyGroup>
     <MicrosoftCodeAnalysisCSharpCodeStyleVersion>$(MicrosoftCodeAnalysisVersion_LatestVS)</MicrosoftCodeAnalysisCSharpCodeStyleVersion>
   </PropertyGroup>
+  <!-- Dependencies from maintenance-packages -->
+  <PropertyGroup>
+    <MicrosoftBclHashCodeVersion>6.0.0</MicrosoftBclHashCodeVersion>
```

Comment:
Why is this PR only changing runtime?  We had discussed that you'll need to change all repos.  Essentially the same as your previous PR but without the conditions - then we selectively apply conditions or modify projects to omit references if they expect them to come from MSBuild.

This is currently failing because you left out the feed with the stable package.
>     /__w/1/s/src/runtime/src/libraries/System.Collections.Immutable/src/System.Collections.Immutable.csproj : error NU1103: Unable to find a stable package System.Runtime.CompilerServices.Unsafe with version (>= 6.1.1) [/__w/1/s/src/runtime/Build.proj]

Reply:
Because I need the PR to start surfacing the failures and then I start fixing the other repos from there.

Reply:
In the previous PR I did a poor job at explaining the reasoning for the change in each commit. I want to be more explicit this time (excluding the first commit).